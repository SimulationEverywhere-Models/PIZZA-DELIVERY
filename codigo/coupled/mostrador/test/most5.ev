00:00:01:000 abrir_m 1
00:00:02:000 atiende_m 2
00:00:08:000 atiende_m 3
00:00:15:000 atiende_m 4
00:00:35:000 recibe_ent_m 3
00:00:45:000 moto_ocupada_m 1
00:01:05:000 atiende_m 5
00:01:10:000 moto_libre_m 1
00:01:15:000 recibe_ent_m 2
00:01:20:000 atiende_m 6
00:01:30:000 atiende_m 7
00:01:55:000 lleno_ent 1
00:02:10:000 recibe_ent_m 4
00:02:15:000 moto_ocupada_m 1
00:02:20:000 lista_ent_m 1
00:02:30:000 moto_libre_m 1
00:02:45:000 recibe_ent_m 6
00:03:00:000 cerrar_m 1
00:03:15:000 recibe_ent_m 7
00:03:30:000 recibe_ent_m 5