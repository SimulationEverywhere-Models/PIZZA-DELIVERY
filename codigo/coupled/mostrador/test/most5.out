00:00:01:000 lista_ll_m 1
00:00:02:000 ocupado_m 1
00:00:04:000 libre_m 1
00:00:04:030 manda_pedido 2
00:00:08:000 ocupado_m 1
00:00:10:000 libre_m 1
00:00:10:030 manda_pedido 3
00:00:15:000 ocupado_m 1
00:00:17:000 libre_m 1
00:00:17:030 manda_pedido 4
00:00:35:000 ocupada_ent_m 1
00:00:35:030 dist_out_m 3
00:00:35:030 libre_m_ent 1
00:01:05:000 ocupado_m 1
00:01:07:000 libre_m 1
00:01:07:030 manda_pedido 5
00:01:15:000 ocupada_ent_m 1
00:01:15:030 dist_out_m 2
00:01:15:030 libre_m_ent 1
00:01:20:000 ocupado_m 1
00:01:22:000 libre_m 1
00:01:22:030 manda_pedido 6
00:01:30:000 ocupado_m 1
00:01:32:000 libre_m 1
00:01:32:030 manda_pedido 7
00:01:55:000 cerrado_m 1
00:02:10:000 ocupada_ent_m 1
00:02:10:030 dist_out_m 4
00:02:10:030 libre_m_ent 1
00:02:20:000 lista_ll_m 1
00:02:45:000 ocupada_ent_m 1
00:02:45:030 dist_out_m 6
00:02:45:030 libre_m_ent 1
00:03:00:000 cerrado_m 1
00:03:15:000 ocupada_ent_m 1
00:03:15:030 dist_out_m 7
00:03:15:030 libre_m_ent 1
00:03:30:000 ocupada_ent_m 1
00:03:30:030 dist_out_m 5
00:03:30:030 libre_m_ent 1
